script/SignFromJson.s.sol

```diff
@@ -27,5 +27,5 @@ contract SignFromJson is MultisigBuilder, JsonTxBuilderBase {
         return vm.envAddress("OWNER_SAFE");
     }
 
-    function _postCheck() internal view override {}
+    function _postCheck() internal view virtual override {}
```

Comment:
From @tynes: 

> If we added a pre hook to go with the post hook then it would work really well with the state diff recording cheatcodes. Thoughts @mds1 @maurelian ?

What exactly would you put in the pre-hook? In general I'm likely to be in favor, but perhaps as a follow up PR to keep a tight scope here?

_(arbitrary comment location to pull that comment into a conversation to ensure it gets addressed before merge)_

Reply:
The pre hook would be the cheatcode for starting state diff recording then the post hook would stop and return the state diffs. I am fine with doing that in a follow up but do think its a good way to force being exhaustive

Reply:
Great, let's track in an issue and resolve this thread once the issue is created

Reply:
issue: https://github.com/ethereum-optimism/superchain-ops/issues/120